### PR TITLE
Improved "disaster restore" doc, added "member update" command descri…

### PR DESCRIPTION
**Pull request by kayrus**
…ption

/cc @robszumski

### CODE REVIEW

**Comment by xiang90** [comment]
can you please separate them into two commits?

**Comment by kayrus** [comment]
Hm, I've used "etcdctl member update" command in updated "disaster restore" doc. Don't how how can I split that.

File: Documentation/admin_guide.md [hunk 1]
```diff
@@ -181,9 +186,9 @@ Once you have verified that etcd has started successfully, shut it down and move
 
 #### Restoring the cluster
 
-Now that the node is running successfully, you should [change its advertised peer URLs](other_apis.md#change-the-peer-urls-of-a-member), as the `--force-new-cluster` has set the peer URL to the default (listening on localhost).
+Now that if the node is running successfully, you should [change its advertised peer URLs](runtime-configuration.md#update-a-member), as the `--force-new-cluster` has set the peer URL to the default (listening on localhost).
 
-You can then add more nodes to the cluster and restore resiliency. See the [runtime configuration](runtime-configuration.md) guide for more details.
+You can then add more nodes to the cluster and restore resiliency. See the [add a new member](runtime-configuration.md#add-a-new-member) guide for more details. **NB:** If you are trying to restore your cluster using old failed etcd nodes, please make sure you have stopped old etcd instances and removed their old data directories specified by the data-dir configuration parameter.
```
Comment (robszumski) [inline Documentation/admin_guide.md]: What does "NB" stand for here?

Comment (kayrus) [inline Documentation/admin_guide.md]: Because it is important when you restore etcd cluster. It is not obvious for users that they have to clean datadir

**Comment by kayrus** [comment]
Any news?

File: Documentation/admin_guide.md (hunk 1 above)
Comment (robszumski) [inline Documentation/admin_guide.md]: Still wondering what the letters "NB" stand for?

**Comment by robszumski** [comment]
We can merge this now (after conflicts are resolved) and we can come back later on and restate the "NB" stuff

**Comment by kayrus** [comment]
Conflict has been resolved.

**Comment by yichengq** [comment]
could you follow to https://github.com/coreos/etcd/blob/master/CONTRIBUTING.md#format-of-the-commit-message to change the commit title and squash into one?

**Comment by kayrus** [comment]
Done

**Comment by kayrus** [comment]
Any news?

**Comment by yichengq** [comment]
I missed this one, and it conflicts with master again. :(
Could you rebase it? i will merge it then

**Comment by kayrus** [comment]
@yichengq Done

File: Documentation/admin_guide.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Using an out-of-date data directory can lead to inconsistency as the member had
 For maximum safety, if an etcd member suffers any sort of data corruption or loss, it must be removed from the cluster.
 Once removed the member can be re-added with an empty data directory.
 
-[members-api]: other_apis.md#members-api
+[remove-a-member]: runtime-configuration.md#remove-a-member
```
Comment (yichengq) [inline Documentation/admin_guide.md]: we need to change where it is used at line 14. or the link is broken.

Comment (yichengq) [inline Documentation/admin_guide.md]: The link doesn't match its name `members API`.

File: Documentation/admin_guide.md [hunk 3]
```diff
@@ -206,7 +212,7 @@ etcd -name infra1 \
 -advertise-client-urls http://10.0.1.13:2379,http://127.0.0.1:2379
 ```
 
-[change peer url]: other_apis.md#change-the-peer-urls-of-a-member
+[change peer url]: runtime-configuration.md#update-a-member
```
Comment (yichengq) [inline Documentation/admin_guide.md]: The link doesn't match its display name `members API`.

**Comment by yichengq** [comment]
LGTM

**Pull request merged**